### PR TITLE
added checkboxes to select/deselect week days

### PR DESCRIPTION
**Pull request by aboubakar20**

### CODE REVIEW

File: src/components/WeekDaysSelector.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template>
+  <div>
+    <h5>Avalable Weeks</h5>
+    <div class="week">
+      <div class="days" v-for="weekDay in weekDays" :key="weekDay.id">
```
Comment (junaidrt) [inline src/components/WeekDaysSelector.vue]: Please `concat` a unique string with `id` to make it true unique in our dom.

File: src/components/WeekDaysSelector.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+<script>
+export default {
+  data() {
+    return {
+      checkedDays: [],
+    };
+  },
+  props: ["weekDays"],
+  updated() {
```
Comment (junaidrt) [inline src/components/WeekDaysSelector.vue]: Please remove this unused hook.

**Review by junaidrt — CHANGES_REQUESTED** [review]

File: src/components/WeekDaysSelector.vue (hunk 1 above)
Comment (aboubakar20) [reply src/components/WeekDaysSelector.vue]: Done

File: src/components/WeekDaysSelector.vue (hunk 2 above)
Comment (aboubakar20) [reply src/components/WeekDaysSelector.vue]: Done

**Review by junaidrt — APPROVED** [review]

**Pull request merged**